### DISCUSSION TITLE
UTM-HV on IpadOS 16.1 keeps crashing after a few minutes of usage.

### DISCUSSION THREAD

**Discussion by Henriouss** [utmapp/UTM#6139] [UTM Help (iOS)]
The VM used is the Ubuntu arm64 found in the UTM library. 
I tried switching up different options for the VM but after running for a few minute it crashes. I can replicate the crash each time by running sudo apt-get update and sudo apt-get upgrade, the VM crashes during the unpacking process.

Win 11 VHDX arm64 has the same behavior even after installing spice tools.

UTM-HV is running on IpadOS 16.1 with trollstore 2 HV enabled on Ipad Pro M1.

[debug.log](https://github.com/utmapp/UTM/files/13674778/debug.log)

**Comment by BokBoy** [comment]
Using too much RAM??

**Comment by Henriouss** [comment]
Setting the VM to even 1GB of RAM and forcing 512mb of JIT cache yields the same results.

**Comment by BokBoy** [comment]
Have you tried another side loading method??

**Comment by Henriouss** [comment]
It's with trollstore 2, using another sideloading method won't enable the HV.

**Comment by BokBoy** [comment]
Ohh... what about another device??

**Comment by Henriouss** [comment]
That's the point of the GitHub issue, to see if it's repeatable on the same configuration as mine.

**Comment by BokBoy** [comment]
> That's the point of the GitHub issue, to see if it's repeatable on the same configuration as mine.

Then that's my end for help...
Sorry we couldn't get this fixed.
